docs/topics/build_setup.md

```diff
@@ -50,6 +51,54 @@ apply plugin: "io.github.takahirom.roborazzi"
 
 </table>
 
+**build.gradle.kts**
+<table>
+<tr><td>plugins</td><td>buildscript</td></tr>
+<tr><td>
+
+Define plugin in root build.gradle.kts
+
+```kotlin
+plugins {
+  ...
+  id("io.github.takahirom.roborazzi") version "[version]" apply false
+}
+```
+
+Apply plugin in module build.gradle.kts
+
+```kotlin
+plugins {
+  ...
+  id("io.github.takahirom.roborazzi")
+}
+```
+
+</td><td>
+
+root build.gradle.kts
+
+```kotlin
+buildscript {
+  dependencies {
+    ...
+    classpath("io.github.takahirom.roborazzi:roborazzi-gradle-plugin:[version]")
+  }
+}
+```
+
+module build.gradle.kts
+
+```kotlin
+plugins {
+    ...
+    id("io.github.takahirom.roborazzi")
+}
+```
+
+</td></tr>
+
+</table>
```

Comment:
Thank you! This document might serve as the first impression of this library. The .kts format has become more standard now. Therefore, I suggest we prioritize the kts format and use a details tag for the build.gradle version. 
How about this?

----
**build.gradle.kts**
<table>
<tr><td>plugins</td><td>buildscript</td></tr>
<tr><td>

Define plugin in root build.gradle.kts

```kotlin
plugins {
  ...
  id("io.github.takahirom.roborazzi") version "[version]" apply false
}
```

Apply plugin in module build.gradle.kts

```kotlin
plugins {
  ...
  id("io.github.takahirom.roborazzi")
}
```

</td><td>

root build.gradle.kts

```kotlin
buildscript {
  dependencies {
    ...
    classpath("io.github.takahirom.roborazzi:roborazzi-gradle-plugin:[version]")
  }
}
```

module build.gradle.kts

```kotlin
plugins {
    ...
    id("io.github.takahirom.roborazzi")
}
```

</td></tr>

</table>
<details>
<summary>
build.gradle version
</summary>
<table>
<tr><td>plugins</td><td>buildscript</td></tr>
<tr><td>

Define plugin in root build.gradle

```groovy
plugins {
  ...
  id "io.github.takahirom.roborazzi" version "[version]" apply false
}
```

Apply plugin in module build.gradle

```groovy
plugins {
  ...
  id 'io.github.takahirom.roborazzi'
}
```

</td><td>

root build.gradle

```groovy
buildscript {
  dependencies {
    ...
    classpath "io.github.takahirom.roborazzi:roborazzi-gradle-plugin:[version]"
  }
}
```

module build.gradle

```groovy
apply plugin: "io.github.takahirom.roborazzi"
```

</td></tr>

</table>

</details>